tests/features/with_test.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import sys
+
+import pytest
+
+from add_trailing_comma._main import _fix_src
+from add_trailing_comma._main import main
+
+
+@pytest.mark.parametrize(
+    ('src', 'plus39'),
```

Comment:
this is a [boolean trap](https://youtu.be/CnRkXO_a5mI) -- either take the actual version or make it a separate test function